crates/executor/src/call.rs

```diff
@@ -36,15 +39,17 @@ pub fn call(
         entry_point_type: starknet_api::deprecated_contract_class::EntryPointType::External,
         entry_point_selector,
         calldata: starknet_api::transaction::Calldata(Arc::new(calldata)),
-        initial_gas: blockifier::abi::constants::INITIAL_GAS_COST,
+        initial_gas: VersionedConstants::latest_constants().gas_cost("initial_gas_cost"),
```

Comment:
I might be misunderstanding since I'm still at the start of my review. But shouldn't this constant be based on the starknet version of the block?

Reply:
It should -- but we should first upgrade to the next blockifier rc so that we actually have the constants for 0.13.0... Mentioned in #1769.